sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/CodeSnippetsAysnc.java

```diff
@@ -0,0 +1,146 @@
+package com.azure.data.tables;
+
+
+import reactor.core.Disposable;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+
+
+public class CodeSnippetsAysnc {
+
+
+    public void AsyncSnippets() {
+
+        //build service client
+        TableServiceAsyncClient tableServiceAsyncClient = new TableServiceAsyncClientBuilder()
+            .connectionString("connectionString")
+            .build();
+
+        //add table
+        Mono<TableAsyncClient> createTableMono = tableServiceAsyncClient.createTable("OfficeSupplies");
+
+        createTableMono.subscribe(Void -> {
+            System.out.println("Table creation successful.");
+        }, error -> {
+            System.out.println("There was an error creating the table. Error: " + error);
+        });
+
+
+
+        //delete a table
+        Mono<Void> deleteTableMono = tableServiceAsyncClient.deleteTable("OfficeSupplies");
+
+        deleteTableMono.subscribe(Void -> {
+            System.out.println("Table deletion successful");
+        }, error -> {
+            System.out.println("There was an error deleting the table. Error: " + error);
+        });
+
+
+        //query tables
+        String selectString = "$select= TableName eq 'OfficeSupplies'";
+        Flux<AzureTable> queryTableFlux = tableServiceAsyncClient.queryTables(selectString);
+        queryTableFlux.subscribe(azureTable -> {
+            System.out.println(azureTable.getName());
+        }, error -> {
+            System.out.println("There was an error querying the service. Error: " + error);
+        });
+    }
+
+    public void InsertEntity(){
```

Comment:
You'll need to format this document. `Ctrl+Alt+L` should reformat this.
The method names should be "insertEntity" rather than "InsertEntity"